.yo-resolve

```diff
@@ -0,0 +1,3 @@
+cli/cli.mjs skip
+generators/server/generator.mjs skip
+README.md skip
```

Comment:
I'm just curious, what does this do?

Reply:
Skip those files when generating.
So they don’t conflict.